defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+ssl_resolvers_v4:
+  - "8.8.8.8"
+ssl_resolvers_v6:
+  - "2001:4860:4860::8888"
+ssl_session_timeout: 1d
+ssl_session_cache: "shared:SSL:50m"
+ssl_protocols: TLSv1.2
+ssl_dhparam: /etc/nginx/dhparam.pem
+ssl_stapling: yes
```

Comment:
I still think this should be 'no' by default since this requires an option to be set in the CSR.